lib/components/user/sub-nav.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Button } from 'react-bootstrap'
+import { LinkContainer } from 'react-router-bootstrap'
+import styled from 'styled-components'
+
+import { ACCOUNT_SETTINGS_PATH, TRIPS_PATH } from '../../util/constants'
+
+const Container = styled.div`
```

Comment:
Any reason why these are not in `styled.js` unlike other similar containers?

Reply:
No, not really. I'll move them. Thanks for flagging.